lib/linter.js

```diff
@@ -232,7 +232,7 @@ class Linter {
         fileConfig,
       });
 
-      let { code } = transform(currentSource, (env) => rule.getVisitor(env));
+      let { code } = transform(currentSource, () => rule.getVisitor());
```

Comment:
technically this is a breaking change, now that it had already been released 😞

Reply:
Should we keep it or should we release this as a bug fix?

Reply:
I'll let @rwjblue decide on this :)

Reply:
I'd just leave it alone, and we can remove for next major. Doesn't really hurt anything to pass the argument...

Reply:
ya, updated 👍